scene/gui/range.cpp

```diff
@@ -373,6 +381,14 @@ bool Range::is_lesser_allowed() const {
 	return shared->allow_lesser;
 }
 
+void Range::set_allow_non_finite(bool p_allow) {
+	shared->allow_non_finite = p_allow;
+}
+
+bool Range::is_non_finited_allowed() const {
```

Comment:
You kept "finited" by accident
```suggestion
bool Range::is_non_finite_allowed() const {
```

Reply:
yeah I will fix that next commit